script/Ciel/Day 10/10_01_CIEL10_8.txt

```diff
@@ -34,7 +34,7 @@ She must have noticed me buzz in through the main gate and come out to greet me.
 -- Page 586, Offset 28719.
 -- 「ただいま琥珀さん。それとごめん。ちょっと私用で遅くなっただけで、具合を悪くしたとかじゃないんです」
 "I'm home, Kohaku-san.
- Sorry I'm so late.
+ Sorry, I'm so late.
```

Comment:
Revert; you wouldn't add a comma in like "Sorry[,] for bothering you", same idea here.